src/structurizr/model/element.py

```diff
@@ -41,9 +42,9 @@ class ElementIO(ModelItemIO, ABC):
     """
 
     name: str = Field(...)
-    description: str = Field("")
-    url: Optional[HttpUrl] = Field(None)
-    relationships: Optional[List[RelationshipIO]] = Field([])
+    description: str = Field(default="")
+    url: Optional[HttpUrl] = Field(default=None)
+    relationships: Optional[List[RelationshipIO]] = Field(default=[])
```

Comment:
`Optional` should be reserved for fields that can be `None`. Using an empty tuple as default is not necessary but I like that it is immutable.

```suggestion
    relationships: List[RelationshipIO] = Field(default=())
```